### DISCUSSION TITLE
Solve gives wrong output.

### DISCUSSION THREAD

**Discussion by xwaeaewcrhomesysplug** [axkr/symja_android_library#250] [Q&A] [answered]
It gives the wrong ans

working prove for math,
k+3/4=k/2
2k+6=4k
6=2k

correct,3
wrong,-3/2

**To Reproduce**

Example input Symja input expression:

```mma
>> Solve(k+3/(4)==(k)/2,{k})
```

Error result:-3

```

```

**Expected behavior**

Expected (correct) result expression:3

```

```

==

**Java**
 - Java15
Win10

**Comment by axkr** [comment, marked as answer]
If you set the paranthesis you get the expected result:

```mma
>> Solve((k+3)/(4)==(k)/2,{k})
{{k->3}}

>> Solve(k+3/(4)==(k)/2,{k})
{{k->-3/2}}
````

**Comment by xwaeaewcrhomesysplug** [comment]
ahhh sorry you are right.will note this down.
